StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -188,8 +192,12 @@ extension PaymentSheet {
                     completion: paymentHandlerCompletion
                 )
             // MARK: ↪ Deferred Intent
-            case .deferredIntent:
-                fatalError("TODO(DeferredIntent)")
+            case .deferredIntent(_, let intentConfig):
+                // Pass in nil for `shouldSavePaymentMethod` since this is a saved payment method
+                intentConfig.confirmHandler(paymentMethod.stripeId,
+                                            nil,
+                                            _handleIntentCreation(_:))
+                completion(.completed) // TODO(porter) Do we want this, move into _handleIntentCreation?
```

Comment:
Seems like we don't want to invoke this until the merchant has invoked their callback telling us processing is complete with a result? thoughts? Currently just invoking this for easier testing.

Reply:
Yes 100%.  See "Confirm flow implementation" here https://docs.google.com/document/d/1htpjRm-3v-R2g2FyfH1lZ3ql_NGm3pgPByzRJfoSs9s/edit#heading=h.qqbklhq8a3bj